src/components/TasksTable/index.jsx

```diff
@@ -92,49 +107,86 @@ export default class TasksTable extends Component {
     });
   };
 
+  handleViewOptionsClick = event => {
+    this.setState({ displayCard: event.target.value === 'true' });
+  };
+
+  renderViewOptions() {
+    return (
+      <Fragment>
+        <IconButton value onClick={this.handleViewOptionsClick}>
+          <TableColumnIcon />
+        </IconButton>
+        <IconButton value={false} onClick={this.handleViewOptionsClick}>
+          <TableLargeIcon />
+        </IconButton>
+      </Fragment>
+    );
+  }
```

Comment:
Instead of having two icons, we can have 1 icon only. When the bugs/issues are displayed in a table format, we show the card icon. When the bugs/issues are displayed in a card format, we show the table icon.